packages/react/src/VideoPlayer/VideoPlayer.tsx

```diff
@@ -0,0 +1,213 @@
+      setPlayedTime(videoRef.current?.currentTime || 0)
+      onTimeUpdate && onTimeUpdate(e)
+    },
+    [onTimeUpdate, videoRef],
+  )
+
+  /* > Play / Pause                                                    */
+  const handleVideoPlayback = () => {
+    if (videoRef.current) {
+      if (videoRef.current.paused) {
+        videoRef.current.play()
+      } else {
+        videoRef.current.pause()
+      }
+    }
+  }
+
+  /* > Hide Default Captions                                                   */
+  useEffect(() => {
+    if (videoRef.current) {
+      for (let i = 0; i < videoRef.current.textTracks.length; i++) {
+        videoRef.current.textTracks[0].mode = 'hidden'
+      }
+    }
+  }, [videoRef])
+
+  /* > Set breakpoint classes                                                   */
+
+  useEffect(() => {
+    const handleResize = () => {
+      const breakpoint = videoWrapperRef.current?.getBoundingClientRect().width
+      if (breakpoint && breakpoint < 650) {
+        videoWrapperRef.current.classList.add(styles.VideoPlayer__breakpointSmall)
+      } else {
+        videoWrapperRef.current?.classList.remove(styles.VideoPlayer__breakpointSmall)
+      }
+    }
+
+    const resizeObserver = new ResizeObserver(_ => {
+      if (videoWrapperRef.current) {
+        handleResize()
+      }
+    })
+
+    handleResize()
+
+    const currentRef = videoWrapperRef.current
+
+    resizeObserver.observe(videoWrapperRef.current as Element)
+    return () => {
+      resizeObserver.unobserve(currentRef as Element)
+    }
+  }, [videoWrapperRef, playing])
+
+  return (
+    <div
+      className={clsx(
+        styles.VideoPlayer__container,
+        styles.VideoPlayer__overlays,
+        !playing && styles.VideoPlayer__showOverlays,
+      )}
+      ref={videoWrapperRef}
+    >
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video
+        ref={videoRef}
+        title={title}
+        controls={false}
+        poster={poster}
+        className={clsx(styles.VideoPlayer, className)}
+        onPlay={handleOnPlay}
+        onPause={handleOnPause}
+        onPlaying={handleOnPlaying}
+        onLoadedMetadata={handleOnLoadedMetadata}
+        onTimeUpdate={handleOnTimeUpdate}
+        {...rest}
+      >
+        {children}
+      </video>
+      {title && (
+        <div className={styles.VideoPlayer__title}>
+          {branding && <MarkGithubIcon size={48} />}
+          <Text size="500" weight="medium" className={styles.VideoPlayer__controlTextColor}>
+            {title}
+          </Text>
+        </div>
+      )}
+      <button className={styles.VideoPlayer__playButton} onClick={handleVideoPlayback}>
+        {!playing && (
+          <span className={styles.VideoPlayer__playButtonInner}>
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              style={{marginLeft: 5}}
```

Comment:
Can we remove this inline style please? Maybe put it in a class on the parent? We also prefer rem over px.

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5